sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOrcScan.scala

```diff
@@ -1647,41 +1648,48 @@ class MultiFileCloudOrcPartitionReader(
       val startingBytesRead = fileSystemBytesRead()
 
       val hostBuffers = new ArrayBuffer[(HostMemoryBuffer, Long)]
+      val filterStartTime = System.nanoTime()
       val ctx = filterHandler.filterStripes(partFile, dataSchema, readDataSchema,
         partitionSchema)
+      val filterTime = System.nanoTime() - filterStartTime
+      val bufferTimeStart = System.nanoTime()
+      var result: HostMemoryBuffersWithMetaDataBase = null
       try {
         if (ctx == null || ctx.blockIterator.isEmpty) {
```

Comment:
the diff is a bit hard to read but can't result = if {...  so we can get rid of the var?

Reply:
Prior code returned earlier (https://github.com/NVIDIA/spark-rapids/pull/6429/files#diff-86813e200aa47eb04dcd8a7bf94491e0dc6d2fa67a09991a2641d947eaf52fc3L1656), so I need to handle that case.

Note that  selecting to hide whitespace from the diff in github makes it much easier to compare. Let me know if you still want me to remove the var, I can probably do something (but not sure if it will fix the diff issue)

Reply:
sorry I was off by one level:
val result = try {.....} and remove the var result

Reply:
@tgravescs tried this. Let me know if that helps.